mlflow/sklearn/__init__.py

```diff
@@ -33,10 +33,10 @@
 from mlflow.utils.autologging_utils import (
```

Comment:
This portion of the change integrates the new `MlflowAutologgingQueueingClient` (defined in `mlflow/utils/autologging_utils/client.py`) into the `mlflow.sklearn` autologging integration.